open-company/open-company-web
# Foc collapsed view

Card: https://trello.com/c/fVmCSLpC

To test TBD

NB: test case where user is loading FoC in collapsed mode with a very high window that fits more than 1 page and make sure all the possible pages are loaded and also the auto load more still works.
